cmd/hack/rpc-trace-compare/main.go

```diff
@@ -59,9 +59,9 @@ func main() {
 			return
 		}
 
-		if err = os.Remove("./traces/" + traceFile); err != nil {
-			fmt.Println(err)
-		}
+		// if err = os.Remove("./traces/" + traceFile); err != nil {
```

Comment:
seems like an unnecessary change for me

Reply:
There is no point to delete traces when comparing the traces. This is undesired side effect.